PowerEditor/src/NppDarkMode.cpp

```diff
@@ -436,6 +459,71 @@ namespace NppDarkMode
 		return g_darkModeSupported;
 	}
 
+	bool isWindowsModeEnabled()
+	{
+		return g_advOptions._enableWindowsMode;
+	}
+
```

Comment:
@donho 
add here and in header, required to properly save "Windows mode" to config
```
void setWindowsMode(bool enable)
{
	g_advOptions._enableWindowsMode = enable;
}
```